include/gatekeeper_lls.h

```diff
@@ -19,6 +19,12 @@
 #ifndef _GATEKEEPER_LLS_H_
 #define _GATEKEEPER_LLS_H_
 
-int run_lls(void);
+struct lls_config {
+	unsigned int	lcore_id;
+};
+
+struct lls_config *alloc_lls_conf(void);
```

Comment:
Shouldn't there be only one instance of LLS? If so, this function should be called `get_lls_conf()` and only return that address.